src/client/ui/dialogs/LoginDialog.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import styles from "./dialog.scss";
+import Button from "../Button";
+import Header from "../Header";
+import StringInput from "../inputs/StringInput";
+
+export default class LoginDialog extends Component {
+  static propTypes = {
+    onLogin: PropTypes.func,
+    authStarted: PropTypes.bool,
+    onCancel: PropTypes.func,
+    hideDialog: PropTypes.func.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: ""
+    };
+  }
+
+  render() {
+    const { onLogin, authStarted, onCancel, hideDialog } = this.props;
+    return (
+      <div className={styles.dialogContainer}>
+        <Header title="Login to Publish" />
+        <div className={styles.loginContainer}>
+          {authStarted ? (
+            <div className={styles.content}>
+              <div className={styles.message}>Email sent! Please click on the link in th email to continue.</div>
```

Comment:
Suggest we add a spinner here maybe?

Reply:
also probably could have a cancel button -- not strictly necessary. if you enter the wrong email address you are stuck basically

Reply:
Hmm, not so sure about the spinner. I think it might be misinterpreted as the app actually doing something that you have to wait for.